src/components/Courses/Courses.js

```diff
@@ -38,19 +38,11 @@ const Courses = ({edges, title}) => {
     setPage(value);
     window.scrollTo(0, 0);
   };
-  const titleText = title ? `Courses: ${title}` : 'Courses';
 
   return (
     <Box m="auto" maxWidth={theme.breakpoints.values.lg}>
-      {title && (
-        <Box className={classes.backToContainer}>
-          <Button component={Link} to="/courses">
```

Comment:
How come you're removing the back button here?

Reply:
The "Courses" link is in the navigation, so having it here a second time is redundant and takes up space. Also, because the "Tags" template mounts this component, it didn't make sense to refer exclusively to Courses when Tags can now link to Courses or Tech Talks